### PR TITLE
Explain examples

### PR DESCRIPTION
**Pull request by hp2500**

### CODE REVIEW

File: examples/explain_example.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""
+Model Explanation
+=================
+"""
+from dabl.models import SimpleClassifier
+
+wine_df = dabl.utils.data_df_from_bunch(load_wine())
```
Comment (amueller) [inline examples/explain_example.py]: undefined name dabl ;) see the CI error.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# notebooks
+explain_examples.ipynb
```
Comment (amueller) [inline .gitignore]: I mean I'm not opposed but I'm not sure this is generally helpful?

Comment (hp2500) [reply .gitignore]: Oh, I thought I had already deleted that. Did it now!

File: examples/explain_example.py (hunk 1 above)
Comment (hp2500) [reply examples/explain_example.py]: Seems like its fixed :)

**Comment by amueller** [comment]
The filename needs to start with ``plot_`` to be run by sphinx gallery. You can check out the documentation build by clicking "doc artifact" in the CI checks to see that it's not run:
https://1170-148831482-gh.circle-artifacts.com/0/doc/auto_examples/explain_example.html

**Comment by amueller** [comment]
Thanks! can you do a train/test split and pass the test part to ``explain``? That way you get way more metrics.

**Comment by amueller** [comment]
Cool! For now this is mostly an example showing that the plots are a bit verbose but I think I have a PR to fix that ;)

**Comment by amueller** [comment]
It's #223, if you like you can check it out, but I might also have a look soon-ish?

**Pull request merged**